mava/systems/ippo/networks.py

```diff
@@ -326,7 +331,10 @@ def critic_fn(inputs: jnp.ndarray) -> networks_lib.FeedForwardNetwork:
         Returns:
             FeedForwardNetwork class
         """
-        critic_network = hk.Sequential(
+        critic_layers = []
+        if observation_network:
+            critic_layers.extend([observation_network])
+        critic_layers.extend(
```

Comment:
Same here

Reply:
So we need extend for line 337, since we have both `MLP_NORM` and `ValueHead`. You need extend to add multiple items.

Reply:
Can we split `MLP_NORM` and `ValueHead` and then append them separately instead of extending? 
This is quite minor, but seems nicer from a consistency standpoint in my opinion.

Reply:
For consistency, I think it is better to have everything as an extends.

Reply:
Sure, either way. All extend or all append :smile: